resource_linode.go

```diff
@@ -106,6 +113,16 @@ func resourceLinodeLinode() *schema.Resource {
 				Optional: true,
 				Default:  true,
 			},
+			"disk_expansion": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+			"swap_size": &schema.Schema{
+				Type: schema.TypeInt,
+				Optional: true,
+				Default: 256,
```

Comment:
I think that it would be better to default to 512mb since that was the size that the provider created before.